tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -1005,6 +1006,10 @@ def test_post_notifications_saves_email_normally_if_save_email_to_queue_fails(cl
         "template_id": template.id,
         "personalisation": {"message": "Dear citizen, have a nice day"}
     }
+
+    # Caching
+    time.sleep(5)
```

Comment:
You can use [freezegun](https://pypi.org/project/freezegun/) as a context manager to control time during tests:
```python
from freezegun import freeze_time

with freeze_time('1999-12-31'):
    pass  # make first call

with freeze_time('2000-01-01'):
    pass  # make second call

Reply:
Thanks! I intend on fixing these tests properly once I fix the lazy loading, and adding a test that auth is eventually consistent